README.md

```diff
@@ -1 +1,71 @@
-# while-and-until-looping-readme
+# The `while` and `until` Constructs
+
+## Objectives
+
+1. Understand the `while` construct and how it implements looping
+2. Understand the `until` looping construct 
+
+## `while`
+
+The `while` construct is a little different from the first two constructs (`loop` and `times`) that we looked at earlier. The `while` construct will keep executing a block as long as a specific condition is `true`.
+
+Remember our long and repetitiously-counting code that used `if` statements to count from `0` to `20`? Well, we can refactor that into simple, readable, *short* code with the `while` construct:
+
+```ruby
+counter = 0
+while counter < 20
+  puts "The current number is less than 20."
+  counter += 1
+end
+```
+
+Think about the above code like this:
+
+*  While it is true that the variable `counter` is set to a value that is less than `20`, execute the code in the block.
+*  Inside the block, `puts` a phrase, and increment the counter by one.
+*  Go back to the top! Check to see if the `counter` is less than `20`. If it is true that the value is less than `20`, go back into the block. Otherwise, break out of the loop and don't execute the code inside the loop.
+
+We can achieve all of that with just a few lines of code utilizing a `while` construct.
+
+## Examples
+
+#### Basic `while` Example: Hot Dog Eating Contest
+
+Let's say you are a world famous competitive eater participating in the Coney Island Nathan's Hot Dog Eating Contest. You're kind of new to the competitive eating game though, so you only have the capacity for seven (7) hot dogs.
+
+
+```ruby
+num_of_hotdogs_eaten = 0
+while num_of_hotdogs_eaten < 7
+  num_of_hotdogs_eaten += 1
+  puts "You have now eaten #{num_of_hotdogs_eaten} hot dog(s)."
+end
+
+puts "You ate a total of #{num_of_hotdogs_eaten} hot dogs!"
+
+# => "You have now eaten 1 hot dog(s)."
```

Comment:
don't use the `=>`. That means return value (check it out in irb). Just do "You have now eaten 2 hot dog(s)"